backend/app/qa_engine.py

```
@@ -1,118 +1,164 @@
+            "If the context doesn't contain enough information, say so.\n\n"
+            f"Context:\n{context}\n\n"
+            f"Question: {question}\n\n"
+            "Answer:"
+        )
+
+        try:
+            if self._use_tensorzero:
+                return self._call_tensorzero(prompt)
+            else:
+                return self._call_ollama(prompt)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)